### PR TITLE
docs(spark-setup): improve namespace caveat description

### PR DESCRIPTION
**Pull request by dargmuesli**
## What are you changing in this pull request and why?

Added details to the namespace caveat description for Spark.

## Checklist
- [x] Review the [Content style guide](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/content-style-guide.md) so my content adheres to these guidelines.
- [x] For [docs versioning](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#about-versioning), review how to [version a whole page](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#adding-a-new-version) and [version a block of content](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#versioning-blocks-of-content).
- [x] Add a checklist item for anything that needs to happen before this PR is merged, such as "needs technical review" or "change base branch."

### CODE REVIEW

File: website/docs/docs/core/connect-data-platform/spark-setup.md [hunk 1]
```diff
@@ -223,6 +225,6 @@ Delta-only features:
 
 ### Default namespace with Thrift connection method
 
-If your Spark cluster doesn't have a default namespace, metadata queries that run before any dbt workflow will fail, causing the entire workflow to fail, even if your configurations are correct. The metadata queries fail there's no default namespace in which to run it.
+A namespace named `default` is required to exist in Spark when connecting via Thrift for dbt to run metadata queries in. You can use Spark's `pyspark` and run `spark.sql("SHOW NAMESPACES").show()` to see the available namespaces and create the required namespace by running `spark.sql("CREATE NAMESPACE default").show()`.
```
Comment (mirnawong1) [inline website/docs/docs/core/connect-data-platform/spark-setup.md]:
```suggestion
To run metadata queries in dbt, you need to have a namespace named `default` in Spark when connecting with Thrift. You can check available namespaces by using Spark's `pyspark` and running `spark.sql("SHOW NAMESPACES").show()`. If the default namespace doesn't exist, create it by running `spark.sql("CREATE NAMESPACE default").show()`.
```

File: website/docs/docs/core/connect-data-platform/spark-setup.md [hunk 2]
```diff
@@ -223,6 +225,6 @@ Delta-only features:
 
 ### Default namespace with Thrift connection method
 
-If your Spark cluster doesn't have a default namespace, metadata queries that run before any dbt workflow will fail, causing the entire workflow to fail, even if your configurations are correct. The metadata queries fail there's no default namespace in which to run it.
+To run metadata queries in dbt, you need to have a namespace named `default` in Spark when connecting with Thrift. You can check available namespaces by using Spark's `pyspark` and running `spark.sql("SHOW NAMESPACES").show()`. If the default namespace doesn't exist, create it by running `spark.sql("CREATE NAMESPACE default").show()`.
 
-To debug, review the debug-level logs to confirm the query dbt is running when it encounters the error: `dbt run --debug` or `logs/dbt.log`.
+If there's a network connection issue instead, your logs will contain `Could not connect to any of [('127.0.0.1', 10000)]` (or similar).
```
Comment (mirnawong1) [inline website/docs/docs/core/connect-data-platform/spark-setup.md]:
```suggestion
If there's a network connection issue, your logs will dispaly an error like `Could not connect to any of [('127.0.0.1', 10000)]` (or something similar).
```

File: website/docs/docs/core/connect-data-platform/spark-setup.md [hunk 3]
```diff
@@ -208,6 +208,8 @@ Spark can be customized using [Application Properties](https://spark.apache.org/
 
 ## Caveats
 
+When facing difficulties you can run `poetry run dbt debug --log-level=debug`. The logs are persisted at `logs/dbt.log`.
```
Comment (mirnawong1) [inline website/docs/docs/core/connect-data-platform/spark-setup.md]:
```suggestion
When facing difficulties, run `poetry run dbt debug --log-level=debug`. The logs are saved at `logs/dbt.log`.
```

**Review by mirnawong1 — APPROVED** [review]
hey @dargmuesli ! thanks for opening this up and happy to merge this for you! made some small tweaks and looks great otherwise!

File: website/docs/docs/core/connect-data-platform/spark-setup.md (hunk 2 above)
Comment (dargmuesli) [reply website/docs/docs/core/connect-data-platform/spark-setup.md]: I think there is a typo in "dispaly"

Comment (mirnawong1) [reply website/docs/docs/core/connect-data-platform/spark-setup.md]: good catch! will change this for you!

**Pull request merged**